pkg/metrics/metrics.go

```diff
@@ -175,13 +178,14 @@ func getLabelsForUnsafeAnnotation(unsafeAnnotation string) prometheus.Labels {
 type MetricDescription struct {
 	FqName string
 	Help   string
+	Type   string
 }
 
 func (hm hcoMetrics) GetMetricDesc() []MetricDescription {
 	res := make([]MetricDescription, len(hm.metricDescList))
 	i := 0
 	for _, md := range hm.metricDescList {
-		res[i] = MetricDescription{FqName: md.fqName, Help: md.help}
+		res[i] = MetricDescription{FqName: md.fqName, Help: md.help, Type: md.mType}
```

Comment:
Not a part of this PR, but you already using the `range` iterator which provides you also the index, which is being discarded by using `_`.
You can assign it to `i`, and then you won't need to: 1) initialize it with 0; 2) increment it by 1 at every iteration.
```
	for i, md := range hm.metricDescList {
		res[i] = MetricDescription{FqName: md.fqName, Help: md.help, Type: md.mType}
	}
```

Same comment goes for line 89 above.

Reply:
Correct me if I'm wrong, but I think in both cases the loops iterate over a map, so `range` provides keys and values.

Reply:
Yes, `hm.metricList` is a map. sorry :(

What do you think about the following code instead (it will relief the need for using an indexing variable):
```
	var res []MetricDescription
	for _, md := range hm.metricDescList {
		res = append(res, MetricDescription{FqName: md.fqName, Help: md.help, Type: md.mType})
	}
```

Reply:
This is just a style nit. I'm good with the current code

Reply:
Agree, it looks better. updated